app/views/community/index.html.slim

```diff
@@ -16,7 +16,7 @@ table.users.table.table-striped.table-bordered.table-condensed.table-hover
       th
       th = sortable :name
       th = sortable :interested_in
-      - if signed_in?
+      - if current_user&.confirmed?
```

Comment:
I think we should make this check be the same `can?` check as you have below. What do you think?

Reply:
🤔To use the `can?` helper , I'd need access to the single user, as I do have below. I'd think this hack is better than alternatives, like adding an extra @user ivar? Or...?
Neat it is not. 😢

Reply:
Hmm, good point. I wonder what people usually do in these circumstances - we are kinda duplicating authorization knowledge here.

Also this seems to be slightly different from what we have specified in the Ability class: from what I understand, confirmed users can: `can :read_email, User, hide_email: false`, can't they? (I may be missing something here! I'm not too familiar with CanCan).

Reply:
That didn't work for me, not as a twoliner, and not as a oneliner :-( 
See next commits.